src/esp/assets/ResourceManager.cpp

```diff
@@ -286,21 +286,46 @@ PhysicsManagerAttributes ResourceManager::loadPhysicsConfig(
   std::string configDirectory =
       physicsFilename.substr(0, physicsFilename.find_last_of("/"));
   // load the rigid object library metadata (no physics init yet...)
-  // NOTE: expect relative paths to the global config
   if (scenePhysicsConfig.HasMember("rigid object paths")) {
     if (scenePhysicsConfig["rigid object paths"].IsArray()) {
       physicsManagerAttributes.setVecStrings("objectLibraryPaths",
                                              std::vector<std::string>());
       for (rapidjson::SizeType i = 0;
            i < scenePhysicsConfig["rigid object paths"].Size(); i++) {
         if (scenePhysicsConfig["rigid object paths"][i].IsString()) {
-          // 1: read the filename (relative path)
-          std::string objPhysPropertiesFilename =
-              configDirectory + "/" +
-              scenePhysicsConfig["rigid object paths"][i].GetString() +
-              ".phys_properties.json";
-          physicsManagerAttributes.appendVecStrings("objectLibraryPaths",
-                                                    objPhysPropertiesFilename);
+          std::string filename =
+              scenePhysicsConfig["rigid object paths"][i].GetString();
+          std::string directory = configDirectory + "/";
```

Comment:
Since you're now using `Directory::join()` to join the paths, I think adding the trailing `/` isn't needed anymore.